plugins/available/base.plugin.bash

```diff
@@ -4,13 +4,24 @@ about-plugin 'miscellaneous tools'
 function ips ()
 {
     about 'display all ip addresses for this host'
+    param 'optional -i'
+    example '$ ips'
+    example '$ ips -i'
     group 'base'
     if command -v ifconfig &>/dev/null
     then
-        ifconfig | awk '/inet /{ gsub(/addr:/, ""); print $2 }'
+        if [ -n "$1" -a "$1" = "-i" ];then
+            ifconfig | grep -B 1 -E "inet "| sed 's/addr://g' | sed 's/: / /g' | awk '{if (NR%3==1) print $1 ":"; else if (NR%3==2) print $2}'
```

Comment:
Beacause running the ifconfig command get different results on different OS.  
on Centos:
lo        Link encap:Local Loopback  
          inet addr:127.0.0.1  Mask:255.0.0.0
          inet6 addr: ::1/128 Scope:Host
          UP LOOPBACK RUNNING  MTU:16436  Metric:1
          RX packets:5302709 errors:0 dropped:0 overruns:0 frame:0
          TX packets:5302709 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:0 
          RX bytes:253434162 (241.6 MiB)  TX bytes:253434162 (241.6 MiB)
On Ubuntu:
lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 390  bytes 29016 (29.0 KB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 390  bytes 29016 (29.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0